<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Optionwindow</class>
 <widget class="QMainWindow" name="Optionwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	color: rgb(0, 123, 255);	
	background-color: rgb(0, 0, 0);

}

QPushButton {
	background-color: rgb(0, 0, 0);
	color: (0, 123, 255);
    border-width: 4px;
	border-color: rgb(0, 123, 255);
	border-style: solid;
	border-radius: 7;
	padding: 3px;
	padding-left: 5px;
	padding-right: 5px;
}

QPushButton:pressed
{
	color: rgb(239, 151, 0);	
	border-color: rgb(239, 151, 0);
}

QProgressBar
{
	background-color: rgb(166, 166, 166);
	color: rgb(0, 0, 0);
    border: 2px rgb(166, 166, 166);
    border-radius: 5px;
}

QProgressBar::chunk {
	border: 2px rgb(166, 166, 166);
    border-top-left-radius: 5px;
    border-bottom-left-radius: 5px;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
	background-color: rgb(0, 123, 255);
   /* width: 40px;
    margin: 0.5px;*/
}

#Labbruch {
	color: rgb(239, 151, 0);
}

#LProgress{
	color: rgb(239, 151, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="button_config">
      <property name="maximumSize">
       <size>
        <width>5000</width>
        <height>300</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Change Config</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="button_back">
      <property name="maximumSize">
       <size>
        <width>5000</width>
        <height>300</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt; Back</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="button_clean">
      <property name="maximumSize">
       <size>
        <width>5000</width>
        <height>300</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Clean Machine</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
